### DISCUSSION TITLE
how would handle multiple window or multiple tab in WebAction Class. Can you please share that part.

### DISCUSSION THREAD

**Discussion by shirshGit** [akshayp7/playwright-typescript-playwright-test#26] [Q&A] [answered]
how would handle multiple window or multiple tab in WebAction Class. Can you please share that part.

**Comment by akshayp7** [comment]
Sure I will try to implement it in next release. Thank you for your feedback.

**Comment by akshayp7** [comment, marked as answer]
> how would handle multiple window or multiple tab in WebAction Class. Can you please share that part.

Have written a new Testcase to Verify for Multiwindow Scenarios. Please refer v0.13.0
